internal/resources/ibm/tg_gateway.go

```diff
@@ -0,0 +1,175 @@
+package ibm
+
+import (
+	"github.com/infracost/infracost/internal/resources"
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+)
+
+// TgGateway struct represents a transit gateway between VPCs
+//
+// Resource information: https://cloud.ibm.com/docs/transit-gateway?topic=transit-gateway-getting-started
+// Pricing information: https://cloud.ibm.com/docs/transit-gateway?topic=transit-gateway-tg-pricing
+type TgGateway struct {
+	Address       string
+	Region        string
+	GlobalRouting bool
+
+	DataTransferLocal  *float64 `infracost_usage:"data_transfer_local"`
```

Comment:
the infracost-usage-example.yml will also need to be updated for any new usage variables that are introduced

Reply:
added